src/Atmos/Model/AtmosModel.jl

```diff
@@ -263,4 +253,42 @@ function init_state!(m::AtmosModel, state::Vars, aux::Vars, coords, t, args...)
   m.init_state(state, aux, coords, t, args...)
 end
 
+
+function numerical_boundary_flux_diffusive!(nf::CentralNumericalFluxDiffusive,
```

Comment:
Maybe we can avoid overloading this by exposing a variant of flux_diffusive! which takes a normal vector

Reply:
Just pushed a slightly different interface. Complication arises from the fact we want to specify BCs in two different ways. I imagine all this could change soon anyway after we have a bit more time to sort out BCs.

Mainly want to get this PR through so that other BCs issues and instability issues can be addressed.

Reply:
Definitely, thanks for wrapping this up.